lib/file-util.js

```diff
@@ -16,9 +16,9 @@ function isDirectory (dir) {
     }
 }
 
-function getFilesToUpload (assetsFolder) {
+function getFilesToUpload (assetsFolder, flatten = false) {
```

Comment:
To avoid a breaking change.

But it's just a number, so noe real problem, I suppose